x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -38028,7 +38028,7 @@
     "xpack.triggersActionsUI.sections.rulesList.rulesListNotifyBadge.openSnoozePanel": "Ouvrir le panneau Répéter",
     "xpack.triggersActionsUI.sections.rulesList.rulesListNotifyBadge.snoozedIndefinitelyTooltip": "Notifications répétées indéfiniment",
     "xpack.triggersActionsUI.sections.rulesList.rulesListSnoozePanel.snoozeFailed": "Impossible de modifier les paramètres de répétition de la règle",
-    "xpack.triggersActionsUI.sections.rulesList.rulesListSnoozePanel.snoozeSuccess": "Répétition de la règle activée",
+    "xpack.triggersActionsUI.sections.rulesList.rulesListSnoozePanel.snoozeSuccess": "Notification de règle de répétition déclenchée par {snoozeDuration}.",
```

Comment:
You do not need to add these translation because at each release, we have a process to rebuild all these translations. So, It is ok to remove it, you only get error when you have duplicate key.